pkg/virtctl/usbredir/usbredir.go

```diff
@@ -58,122 +58,147 @@ type usbredirCommand struct {
 	clientConfig clientcmd.ClientConfig
 }
 
-func (usbredirCmd *usbredirCommand) Run(command *cobra.Command, args []string) error {
-	if _, err := exec.LookPath(usbredirClient); err != nil {
-		return fmt.Errorf("Error on finding %s in $PATH: %s", usbredirClient, err.Error())
-	}
+type ClientConnectFn func(device, address string) error
 
-	namespace, _, err := usbredirCmd.clientConfig.Namespace()
-	if err != nil {
-		return err
-	}
+type Client struct {
```

Comment:
218da3e68a3849439c6f8eb0199b3cca7e08ebfc would be better to create `client.go` and move it there, now it is hard to review.

Reply:
Ah, sure, I can do that